### PR TITLE
refactor: load CSS variables via global import instead of using `<ThemeProvider>`

### PR DESCRIPTION
**Pull request by emmenko**
Temporary workaround for #1410

### CODE REVIEW

File: packages/gatsby-theme-docs/globals.css [hunk 1]
```diff
@@ -0,0 +1,6 @@
+@import '@commercetools-uikit/design-system/materials/custom-properties.css';
+
+:root {
+  --font-family-default: 'Roboto', sans-serif;
+  --font-family-body: 'Roboto Mono', monospace;
+}
```
Comment (emmenko) [inline packages/gatsby-theme-docs/globals.css:6]: Overriding some design tokens here.

File: packages/ui-kit/src/components/globals.tsx [hunk 2]
```diff
@@ -2,10 +2,17 @@ import React from 'react';
...
+type TGlobalsProps = {
+  websitePrimaryColor: string;
+};
+
+const Globals = (props: TGlobalsProps) => (
   <Global
     styles={css`
+      :root {
+        --website-primary-color: ${props.websitePrimaryColor};
+      }
```
Comment (emmenko) [inline packages/ui-kit/src/components/globals.tsx:14]: This is defined here as it depends on the prop.

**Review by kark — APPROVED** [review]
👍

**Comment by emmenko** [comment]
@commercetools/docs-tech For awareness

**Pull request merged**